client/src/templates/Challenges/components/independent-lower-jaw.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+import { useTranslation } from 'react-i18next';
+import { Button } from '@freecodecamp/ui';
+import { Test } from '../../../redux/prop-types';
+import { challengeTestsSelector } from '../redux/selectors';
+import { isSignedInSelector } from '../../../redux/selectors';
+import { apiLocation } from '../../../../config/env.json';
+import { openModal, submitChallenge, executeChallenge } from '../redux/actions';
+import Help from '../../../assets/icons/help';
+import callGA from '../../../analytics/call-ga';
+
+import './independent-lower-jaw.css';
+import Reset from '../../../assets/icons/reset';
+
+const mapStateToProps = createSelector(
+  challengeTestsSelector,
+  isSignedInSelector,
+  (tests: Test[], isSignedIn: boolean) => ({
+    tests,
+    isSignedIn
+  })
+);
+
+const mapDispatchToProps = {
+  openHelpModal: () => openModal('help'),
+  openResetModal: () => openModal('reset'),
+  executeChallenge,
+  submitChallenge
+};
+
+interface IndependentLowerJawProps {
+  openHelpModal: () => void;
+  openResetModal: () => void;
+  executeChallenge: () => void;
+  submitChallenge: () => void;
+  tests: Test[];
+  isSignedIn: boolean;
+}
+export function IndependentLowerJaw({
+  openHelpModal,
+  openResetModal,
+  executeChallenge,
+  submitChallenge,
+  tests,
+  isSignedIn
+}: IndependentLowerJawProps): JSX.Element {
+  console.log(isSignedIn, 'isSignedIn in IndependentLowerJaw');
+  const { t } = useTranslation();
+  const firstFailedTest = tests.find(test => !!test.err);
+  const hint = firstFailedTest?.message;
+  const [showHint, setShowHint] = React.useState(false);
+  const [showSubmissionHint, setShowSubmissionHint] = React.useState(true);
+
+  const isChallengeComplete = tests.every(test => test.pass);
+
+  React.useEffect(() => {
+    setShowHint(!!hint);
+  }, [hint]);
+
+  const isMacOS = navigator.userAgent.includes('Mac OS');
+  const checkButtonText = isMacOS ? t('⌘ + Enter') : t('ctrl-enter');
```

Comment:
This looks like it was changed

```suggestion
  const checkButtonText = isMacOS ? t('command-enter') : t('ctrl-enter');
```